Bulldozer.BinaryFile/Maps/GroupImage.cs

```diff
@@ -54,7 +54,7 @@ public int Map( ZipArchive folder, BinaryFileType groupImageType )
             var emptyJsonObject = "{}";
             var newFileList = new List<GroupDocumentKeys>();
             var importedGroups = new GroupService(lookupContext)
-                .Queryable().AsNoTracking().Where(g => g.ForeignKey != null && g.ForeignKey.StartsWith("G_"));
+                .Queryable().AsNoTracking().Where(g => g.ForeignKey != null && g.ForeignKey.StartsWith( importInstanceFKPrefix + "^G_"));
```

Comment:
code formatting, might want to run it on the document just in case.
```suggestion
                .Queryable().AsNoTracking().Where(g => g.ForeignKey != null && g.ForeignKey.StartsWith( importInstanceFKPrefix + "^G_" ));
```